src/main/scala/xerial/sbt/pack/PackPlugin.scala

```diff
@@ -355,14 +357,16 @@ object PackPlugin extends AutoPlugin with PackArchive {
       def linkToScript(name: String) =
         "\t" + """ln -sf "../$(PROG)/current/bin/%s" "$(PREFIX)/bin/%s"""".format(name, name)
 
-      // Create Makefile
-      // TODO Use custom template (packMakefileTemplate)
-      val makefile = {
-        val additionalScripts = binScriptsDir.flatMap(_.listFiles).map(_.getName)
-        val symlink = (mainTable.keys ++ additionalScripts).map(linkToScript).mkString("\n")
-        LaunchScript.generateMakefile(name.value, symlink)
+      if (packGenerateMakefile.value) {
```

Comment:
In sbt-1.0, we need to evaluate task outside the conditional branch (e.g., if-else block). 
So could you rewrite this like
```scala
val projectName = name.value
val makefile = { ...
```

Reply:
Thanks a lot @xerial , I'll push a commit soon.